tools/create_operators_pull_request.py

```diff
@@ -0,0 +1,245 @@
+    )
+    parser.add_argument(
+        "--operator-name",
+        type=str,
+        required=True,
+        help="The name of the KUDO operator (e.g., cassandra, kafka, spark)",
+    )
+    parser.add_argument(
+        "--operator-git-tag",
+        type=str,
+        required=True,
+        help="The git tag in the KUDO operator repository to bring the "
+        + "files from. This is also the directory name in "
+        + "kudobuilder/operators, e.g., repository/cassandra/$OPERATOR_GIT_TAG",
+    )
+    parser.add_argument(
+        "--github-token",
+        type=str,
+        required=True,
+        help="The GitHub token used for opening the PR against kudobuilder/operators",
+    )
+    parser.add_argument(
+        "--git-commit-message",
+        type=str,
+        help="The git commit message in the kudobuilder/operators PR branch. "
+        + "This will also be the PR title",
+    )
+    parser.add_argument(
+        "--operators-base-branch",
+        type=str,
+        default="master",
+        help="The kudobuilder/operators branch to open a PR against",
+    )
+    parser.add_argument(
+        "--git-user",
+        type=str,
+        default="git",
+        help="The git user for cloning and pushing via SSH",
+    )
+    parser.add_argument(
+        "--debug",
+        action="store_true",
+        default=False,
+        help="Show debug output from all operations performed",
+    )
+
+    args = parser.parse_args()
+    operator_repository = args.operator_repository
+    operator_name = args.operator_name
+    operator_git_tag = args.operator_git_tag
+    github_token = args.github_token
+    git_commit_message = args.git_commit_message or (
+        f"Release {operator_name} {operator_git_tag} (automated commit)."
+    )
+    operators_base_branch = args.operators_base_branch
+    git_user = args.git_user
+    debug = args.debug
+
+    base_directory = tempfile.mkdtemp("_kudo_operator_tools")
+    operators_repository = KUDOBUILDER_OPERATORS_REPOSITORY
+    operators_repository_url = (
+        f"{git_user}@github.com:{operators_repository}.git"
+    )
+    operators_branch = (
+        f"{operator_name}_{operator_git_tag}_{random_short_string()}"
+    )
+
+    rc, operators_directory, error_message = clone_repository(
+        operators_repository_url, operators_base_branch, base_directory, debug
+    )
+    if rc != 0:
+        log.error(error_message)
+        return rc
+
+    operator_repository_url = (
+        f"https://{git_user}:{github_token}@github.com"
+        + f"/{operator_repository}.git"
+    )
+
+    rc, operator_directory, error_message = clone_repository(
+        operator_repository_url, operator_git_tag, base_directory, debug
+    )
+    if rc != 0:
+        log.error(error_message)
+        return rc
+
+    rc, stdout, stderr = run(
+        f"cd {operators_directory} && git checkout -b {operators_branch}",
+        debug=debug,
+    )
+    if rc != 0:
+        return (
```

Comment:
This function is supposed to return just `rc`.

Reply:
Which function?

Reply:
`utils.run` returns `rc, stdout, stderr`.